py_zipkin/thrift/__init__.py

```diff
@@ -13,6 +13,7 @@
 thrift_filepath = os.path.join(os.path.dirname(__file__), 'zipkinCore.thrift')
 zipkin_core = thriftpy.load(thrift_filepath, module_name="zipkinCore_thrift")
 
+SERVER_ADDR_VAL = '\x01'
```

Comment:
i tested it in py3 and both values result in the same thriftpy object in the end.